tests/envs/dummy_env.py

```diff
@@ -0,0 +1,17 @@
+import gym
+
+
+class DummyEnv(gym.Env):
+    def __init__(self):
+        super().__init__()
+        self.act_space = gym.spaces.Discrete(2)
+        self.obs_space = gym.spaces.Discrete(2)
+
+    def reset(self):
+        return self.obs_space.sample()
+
+    def step(self, action):
+        return self.obs_space.sample(), 0, False, {}
+
+
+gym.register("Dummy-v0", entry_point="tests.envs.dummy_env:DummyEnv")
```

Comment:
Do we need the full path here? (`tests.envs.dummy_env`) Or can we just do `dummy_env:DummyEnv`?

Reply:
@arjun-kg Could you test this locally with the example from #2809 `pybullet_envs:HopperBulletEnv-v0`? Does this work with it?

Reply:
@pseudo-rnd-thoughts We actually discussed this exact thing on Discord, and it turns out that pybullet tried to be too clever for its own good, so its registration method fails with the new registration setup. I'm currently working on adding a compatibility layer that would be only slightly cursed, but would deprecate their approach in a more "chill" way instead of breaking everything.